test_conformance/math_brute_force/main.cpp

```diff
@@ -1161,7 +1161,11 @@ test_status InitCL( cl_device_id device )
     }
 
     gDevice = device;
-
+    test_status spirv_status;
+    spirv_status = check_spirv_compilation_readiness(device);
```

Comment:
Is there any reason we can't do that for all suites in the harness code?

Reply:
Ok , done